packages/dropdowns/src/Menu/Menu.js

```diff
@@ -64,9 +83,7 @@ const Menu = props => {
             };
           }
 
-          if (isOpen) {
-            scheduleUpdate();
```

Comment:
So the horizontal display issue is happening as this has been removed now. 

The `computedStyles` below gives popper a smaller div to position that doesn't reflect the actual size of the menu and its contents, scheduleUpdate in popper uses rAF to get the coords before paint but now that we use `useEffect` it triggers after paint/layout so the first time the menu is opened it's in the wrong position, after that it's fine.

If we change the `computedStyles` to be:

```js
computedStyles = {
  visibility: 'hidden',
  position: 'absolute'
}
```

It gets in a better position but is still off horizontally on first render.

Reply:
Actually if we just drop the `position: absolute` this works fine now.

Reply:
If we only leave `visibility: hidden` then the invisible menu will block interactions of content below it when closed. This also removes the animation.

I'll work on this some more locally.

Reply:
@jzempel and I were able to resolve this locally. The trick was to apply this positioning against the element that popper is positioning against, rather than the `Menu` directly.

I also apply the `animate` prop (`is-open` className) on open to enable the animations start during initial opening.